packages/cf.js/API_REFERENCE.md

```diff
@@ -10,22 +10,24 @@
     - Lifecycle
         - `on(eventType, callback: Function)`
             - eventTypes
-                - `proposeInstall(proposal: {appInstanceId, appDefinition, terms}, function reject())`
                 - `install(appInstance)`
-                - `rejectInstall(proposal: {appInstanceId, appDefinition, terms})`
+                - `rejectInstall(appInstance)`
```

Comment:
@mvanderh @patience-tema-baron Is there are reason we offer an `appInstance` on the `rejectInstall` event? Isn't the `appInstance` created *after* `install`?